includes/ApplHandlingByStr.php

```diff
@@ -49,14 +49,14 @@ function showStatus( $stat_str )
 function showFullStatus( $stat_str )
 {
 	$STR = explode ( ';', $stat_str );
-	for ( $i =  1;  $i < count ( $STR ) ; $i++ )
+	for ( $i =  0;  $i < count ( $STR ) ; $i++ )
 	{
 		$str = explode ( ',', $STR[$i] );
-		$usr = $str[0];
+		
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->